lib/cdo/delete_accounts_helper.rb

```diff
@@ -97,12 +103,24 @@ def self.purge_orphaned_students(user_id)
     end
   end
 
-  # Remove all user generated content associated with any PD the user has been through.
+  # Remove all user generated content associated with any PD the user has been through, as well as
+  # all PII associated with any PD records.
   # @param [Integer] The ID of the user to clean the PD content.
   def self.clean_and_destroy_pd_content(user_id)
     PeerReview.where(reviewer_id: user_id).each(&:clear_data)
 
     Pd::TeacherApplication.where(user_id: user_id).each(&:destroy)
+    Pd::FacilitatorProgramRegistration.where(user_id: user_id).each(&:clear_form_data)
+    Pd::RegionalPartnerProgramRegistration.where(user_id: user_id).each(&:clear_form_data)
+    Pd::WorkshopMaterialOrder.where(user_id: user_id).each(&:clear_form_data)
```

Comment:
This one is not a form. It's [clear_data](https://github.com/code-dot-org/code-dot-org/blob/5e545f1cbad96efce48bab1456a675d7bcd9b7c9/dashboard/app/models/pd/workshop_material_order.rb#L198)

Reply:
Done. Sorry for sloppiness.

Reply:
No worries! That's what reviews are for. You've certainly caught plenty similar errors of mine :)